payments/serializers/payments.py

```
@@ -0,0 +1,54 @@
+from authentication.serializers import UserSerializer
+from instagram.models import User as InstagramUser
+from payments.models import GatewayOption
+from payments.models import Payment
+
+
+class PaymentListSerializer(serializers.ModelSerializer):
+    user = UserSerializer(read_only=True)
+
+    class Meta:
+        model = Payment
+        fields = ["id", "amount", "status", "created_at", "user", "reference_type"]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)